README.md

```diff
@@ -112,7 +112,7 @@ Various inputs are defined in [`action.yml`](action.yml) to let you configure th
 | Name | Description | Default |
 | - | - | - |
 | `repo-token` | Token to use to authorize label changes. Typically the GITHUB_TOKEN secret, with `contents:read` and `pull-requests:write` access | `github.token` |
-| `configuration-path` | The path to the label configuration file | `.github/labeler.yml` |
+| `configuration-path` | The path to the label configuration file. If not found in the current working directory, will read from the source repository via the Github API. | `.github/labeler.yml` |
```

Comment:
Can there be a "full" usage example for the new feature? How the labeler.yml will be in the current directory and/or `.github`?

Reply:
Honestly don't see how that's necessary. It's a path to a file, why would that need an example?

Reply:
To give an idea to the reader how they would use this feature. As a reader I'll be wondering: How would I get the configuration file in the machine running the workflow? Is there a best practice to that matter?